### PR TITLE
Publish coq-antivalence.1.0.0

### PR DESCRIPTION
**Pull request by ivanbakel**
Inspired by [this StackOverflow answer](https://stackoverflow.com/questions/61289868/type-inequality-without-cardinality-arguments/61292226#61292226), `coq-antivalence` is a plugin that generates type inequality axioms. This lets users effectively treat the set of inductive definitions in scope as an inductive definition itself, including performing `inversion`-style proofs on polymorphic types.

Please see the [project main page](https://github.com/ivanbakel/coq-antivalence) for more details.

I would really appreciate any suggested improvements with regards to the package keywords or the category.

### CODE REVIEW

**Comment by ivanbakel** [comment]
Since the Coq package submission guide suggested that the Ocaml code be reviewed - it can be found in the linked Github repo, particularly in [this file](https://github.com/ivanbakel/coq-antivalence/blob/master/src/type_inequality.ml) which contains the plugin logic.

File: released/packages/coq-antivalence/coq-antivalence.1.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  [make "install"]
+]
+
+url {
+  src: "https://github.com/ivanbakel/coq-antivalence/archive/1.0.0.tar.gz"
+  checksum: "sha256=a367cf9cd74b5fc644a23a802e695646ff688869363812295472f9baba60021f"
+}
+
+tags: [
+  "category:Miscellaneous/Coq Extensions"
```
Comment (clarus) [inline released/packages/coq-antivalence/coq-antivalence.1.0.0/opam:35]:
```suggestion
  "category:Miscellaneous/Coq Extensions"
  "logpath:Antivalence"
```
The install directory for `Require Import`.

**Comment by clarus** [comment]
Thanks for this package on type inequalities, I am not able to review the OCaml code but for me this is OK to merge.

**Comment by gares** [comment]
> I am not able to review the OCaml code but for me this is OK to merge.

the code looks OK

**Pull request merged**